packages/database/src/core/snap/Node.ts

```diff
@@ -119,7 +119,7 @@ export interface Node {
    * @param {boolean=} exportFormat True for export format (also wire protocol format).
    * @return {*} Value of this node as JSON.
    */
-  val(exportFormat?: boolean): Object;
+  val(exportFormat?: boolean): any;
```

Comment:
Can we use lowercase `object` here?

Reply:
It [can be a primitive value](https://github.com/firebase/firebase-js-sdk/blob/ae32b2b58a3969266ab701a6b2a4895b790c8ea8/packages/database/src/core/snap/LeafNode.ts#L189). The pre-TS types in the comments confirm this as well. Sometimes the best type to use is `any`. :man_shrugging:

Reply:
At least internally, for cases like this we have migrated to `unknown`, which enforces type safety in the calling function.